<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetworkPagePPPoe</class>
 <widget class="QWidget" name="NetworkPagePPPoe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="1" column="0">
       <widget class="QLabel" name="label_dynamicIp">
        <property name="text">
         <string>Dynamic IP</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="MyLineEditIP" name="lineEdit_dynamicIp">
        <property name="text">
         <string>0.0.0.0</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_username">
        <property name="text">
         <string>User Name</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_password">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_pppoe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>PPPoE</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="ComboBox" name="comboBox_pppoe">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>4</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_confirm">
        <property name="text">
         <string>Confirm Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="MyLineEdit" name="lineEdit_username">
        <property name="maxLength">
         <number>63</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="MyLineEdit" name="lineEdit_password">
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="MyLineEdit" name="lineEdit_confirm">
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_note">
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Note:If both UPnP and PPPoE are enabled, only PPPoE will take effect.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="style" stdset="0">
      <string>blue_text</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_apply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEditIP</class>
   <extends>QLineEdit</extends>
   <header>mylineeditip.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
